internal/scriptrun/scriptrun.go

```diff
@@ -135,12 +135,14 @@ func (s *ScriptRun) Run(script *project.Script, args []string) error {
 
 	if lang == language.Unknown {
 		if len(attempted) > 0 {
-			return locale.NewInputError(
-				"err_run_unknown_language_fallback",
-				"The language for this script is not supported or not available on your system. Attempted script execution with: {{.V0}}. Please configure the 'language' field with an available option (one, or more, of: {{.V1}})",
-				strings.Join(attempted, ", "),
-				strings.Join(language.RecognizedNames(), ", "),
-			)
+			return errs.AddTips(
```

Comment:
We should only show this if `allowThirdParty=true`; http://github.com/ActiveState/cli/blob/9f570a0bdce116650606e8cfcc04cb9ecac3b64c/internal/language/language.go#L74-L74

There's no point in showing this tip if someone is using a platform provided language.